contracts/ThalesRoyale/ThalesRoyalePrivateRoom.sol

```diff
@@ -199,46 +278,56 @@ contract ThalesRoyalePrivateRoom is Owned, Pausable {
         roundStartTimeInRoom[_roomNumber] = block.timestamp;
         roundEndTimeInRoom[_roomNumber] = roundStartTimeInRoom[_roomNumber] + roundLengthInRoom[_roomNumber];
         totalPlayersInARoomInARound[_roomNumber][1] = numberOfPlayersInRoom[_roomNumber];
+        unclaimedRewardPerRoom[_roomNumber] = rewardPerRoom[_roomNumber];
 
-        emit RoyaleStartedForRoom(_roomNumber);
+        emit RoyaleStartedForRoom(_roomNumber, numberOfPlayersInRoom[_roomNumber], rewardPerRoom[_roomNumber]);
     }
 
     function takeAPositionInRoom(uint _roomNumber, uint _position) external onlyRoomParticipantes(_roomNumber) {
-        require(_position == 1 || _position == 2, "Position can only be 1 or 2");
+        require(_position == DOWN || _position == UP, "Position can only be 1 or 2");
         require(roomStarted[_roomNumber], "Competition not started yet");
         require(!roomFinished[_roomNumber], "Competition finished");
-        require(positionInARoundPerRoom[_roomNumber][msg.sender][currentRoundInRoom[_roomNumber]] != _position, "Same position");
+        require(
+            positionInARoundPerRoom[_roomNumber][msg.sender][currentRoundInRoom[_roomNumber]] != _position,
+            "Same position"
+        );
 
-         if (currentRoundInRoom[_roomNumber] != 1) {
+        if (currentRoundInRoom[_roomNumber] != 1) {
             require(isPlayerAliveInASpecificRoom(msg.sender, _roomNumber), "Player no longer alive");
         }
 
-        require(block.timestamp < roundStartTimeInRoom[_roomNumber] + roundChoosingLengthInRoom[_roomNumber], "Round positioning finished");
+        require(
+            block.timestamp < roundStartTimeInRoom[_roomNumber] + roundChoosingLengthInRoom[_roomNumber],
+            "Round positioning finished"
+        );
 
         // this block is when sender change positions in a round - first reduce
-        if(positionInARoundPerRoom[_roomNumber][msg.sender][currentRoundInRoom[_roomNumber]] == 1){
-            positionsPerRoundPerRoom[_roomNumber][currentRoundInRoom[_roomNumber]][1]--;
-        }else if (positionInARoundPerRoom[_roomNumber][msg.sender][currentRoundInRoom[_roomNumber]] == 2) {
-            positionsPerRoundPerRoom[_roomNumber][currentRoundInRoom[_roomNumber]][2]--;
+        if (positionInARoundPerRoom[_roomNumber][msg.sender][currentRoundInRoom[_roomNumber]] == DOWN) {
+            positionsPerRoundPerRoom[_roomNumber][currentRoundInRoom[_roomNumber]][DOWN]--;
```

Comment:
Please use `.add()` and `.sub()` for math operation. 
The math library is imported, and protects from overflows.

Reply:
done. 👍